examples/bmp280_normal_mode.py

```diff
@@ -0,0 +1,35 @@
+"""
+Example showing how the BMP280 library can be used to set the various
+parameters supported by the sensor.
+Refer to the BMP280 datasheet to understand what these parameters do
+"""
+import time
+
+import board
+import busio
+import adafruit_bmp280
+
+# Create library object using our Bus I2C port
+i2c = busio.I2C(board.SCL, board.SDA)
+bmp280 = adafruit_bmp280.Adafruit_BMP280_I2C(i2c)
+
+# OR create library object using our Bus SPI port
+#spi = busio.SPI(board.SCK, board.MOSI, board.MISO)
+#bme_cs = digitalio.DigitalInOut(board.D10)
+#bme280 = adafruit_bme280.Adafruit_BME280_SPI(spi, bme_cs)
```

Comment:
Looks like this was left as bme280. Just need to change it to:
```
bmp_cs = digitalio.DigitalInOut(board.D10)
bmp280 = adafruit_bmp280.Adafruit_BMP280_SPI(spi, bmp_cs)
```

Reply:
Good catch.  Fixed that with commit b9e41db49